resources/memcached_instance.rb

```diff
@@ -17,16 +18,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+include Memcached::Helpers
 
-provides :memcached_instance_sysv_init
-
-provides :memcached_instance, os: 'linux'
+resource_name :memcached_instance
```

Comment:
you probably want to also provide the legacy name so things don't break for all the users of the systemd resource

Reply:
Is this done by setting ``provides``? If so I'm having trouble getting the tests to pass by doing that. It seems as though it's not touching the ``systemd_unit`` resource. Is there a better way around this?

Reply:
Here's an example of a backwards compat name
https://github.com/chef-cookbooks/cron/blob/master/resources/access.rb#L26